pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -1140,8 +1151,8 @@ void Panels::MemoryCardListPanel_Simple::ReadFilesAtMcdFolder()
 	for (uint i = 0; i < memcardList.size(); i++)
 	{
 		McdSlotItem currentCardFile;
-		bool isOk = EnumerateMemoryCard(currentCardFile, memcardList[i], m_FolderPicker->GetPath());
-		if (isOk && !isFileAssignedAndVisibleOnList(currentCardFile.Filename))
+		bool isOk=EnumerateMemoryCard( currentCardFile, memcardList[i], m_FolderPicker->GetPath() );
+		if( isOk && !isFileAssignedAndVisibleOnList( wxFileName(Path::ToWxString(currentCardFile.Filename)) ) )
```

Comment:
Ditto.

Reply:
I ran it through the clang formatter. If it's wrong just unresolve and let me know what's up